shtdlib.sh

```diff
@@ -84,10 +85,13 @@ if [ "${os_family}" == 'RedHat' ]; then
         os_name='redhat';
     fi
 elif [ "${os_family}" == 'Debian' ]; then
+    os_codename="$(grep UBUNTU_CODENAME /etc/os-release | awk -F= '{print $2}')"
```

Comment:
Do we need to handle any cases where this file does not exist or is empty?

Reply:
Updated